app/services/proceeding_types_service.rb

```diff
@@ -13,9 +13,32 @@ def add(proceeding_type_id: nil, scope_limitation: nil)
     end
   end
 
+  def remove(proceeding_type_id: nil, scope_limitation: nil)
+    ActiveRecord::Base.transaction do
+      application_proceeding_type(proceeding_type_id)&.destroy!
+      @legal_aid_application.clear_scopes!
+      AddScopeLimitationService.call(@legal_aid_application, scope_limitation) if application_proceeding_types.any?
```

Comment:
I don't think you need this line
      `AddScopeLimitationService.call(@legal_aid_application, scope_limitation) if application_proceeding_types.any?`
      and even the parameter `, scope_limitation: nil`
      
      since you're not adding anything here just removing . Removing scope limitation will be a different jira ticket